js/mailbridge.js

```diff
@@ -138,10 +139,15 @@ MailBridge.prototype = {
   __receiveMessage: function mb___receiveMessage(msg) {
     var implCmdName = '_cmd_' + msg.type;
     if (!(implCmdName in this)) {
-      this._LOG.badMessageType(msg.type);
+      logic(this, 'badMessageType', { type: msg.type });
       return;
     }
-    var rval = this._LOG.cmd(msg.type, this, this[implCmdName], msg);
+    logic(this, 'cmd', {
```

Comment:
uh, and I did scan for other "calls" cases, and I think they were all in the loggest test framework, so this is the only call-site that matters.